bokeh/colors/util.py

```diff
@@ -119,12 +119,40 @@ def __init__(self, name: str, r: int, g: int, b: int) -> None:
         self.name = name
         super().__init__(r, g, b)
 
+    @classmethod
+    def find(cls, name: str) -> NamedColor:
+        # If name not recognised, raises ValueError
+        index = cls.__all__.index(name)
+        return cls.colors[index]
```

Comment:
This `ValueError` won't be meaningful without knowledge of the implementation. This should either catch the exception and throw a meaningful one, or preferably return `None` indicate unknown color. My preference is to return `None`, as not finding a color is not an exceptional situation, though this understanding varies greatly across Python's APIs.

Reply:
Now returning `None` if color name is not recognised.